README.md

```diff
@@ -91,10 +91,12 @@ Such questions arise frequently in customer segmentation (what is the effect of
 
 # News
 
-**September 4, 2020:** Release v0.8.0b1, see release notes [here](https://github.com/Microsoft/EconML/releases/tag/v0.8.0b1)
+**November 16, 2020:** Release v0.7.0, see release notes [here](https://github.com/Microsoft/EconML/releases/tag/v0.8.0)
```

Comment:
needs to update the dates, and the version should be v0.8.0?

Reply:
Yes, good catch, I've updated accordingly.